ProcessLib/SmallDeformation/SmallDeformationProcessData.h

```diff
@@ -33,6 +33,7 @@ struct SmallDeformationProcessData
 {
     SmallDeformationProcessData(
         MeshLib::PropertyVector<int> const* const material_ids_,
+        std::vector<int>&& deactivated_material_subdomains_,
```

Comment:
Deactivation of subdomains looks like a very general feature. IMHO the proper solution would be to implement this feature on a higher level for all processes, not specifically for SD.
The current PR is a good starting point, but one should not forget future extensions.
Or is that feature only application relevant for SD?

Reply:
will move it to Process.h

Reply:
or to ProcessData